scripts/process_jartic_parallel.py

```
@@ -0,0 +1,320 @@
+
+
+def process_archive_parallel(archive_path: Path, num_workers: int = None):
+    manager = ExternalDataManager()
+    processed_path = manager.external_data_path / 'jartic' / 'processed'
+    processed_path.mkdir(parents=True, exist_ok=True)
+    
+    year_month = archive_path.stem.replace('jartic_typeB_', '')
+    output_file = processed_path / f"jartic_traffic_{year_month}.csv"
+    
+    if output_file.exists():
+        logger.info(f"Removing existing file: {output_file}")
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)